src/RepoAuditor/EntryPoint.py

```diff
@@ -34,7 +34,7 @@
 # ----------------------------------------------------------------------
 class NaturalOrderGrouper(TyperGroup):
     # ----------------------------------------------------------------------
-    def list_commands(self, *args, **kwargs):
+    def list_commands(self, *_args, **_kwargs):

```

Comment:
I'm guessing that adding an underscore prefix disables the `ruff` warning? If so, I think that I'd rather explicitly disable the warning as that will probably be clearly to future readers. But, let's discuss in more detail if you have a strong opinion otherwise.

Reply:
I would prefer to disable the warning. Disabling seemed like a bit of a cop-out which is why I wanted your feedback before I did it.